Exercises/tools_minimd/CMakeLists.txt

```
@@ -1,36 +1,40 @@
 cmake_minimum_required(VERSION 3.16)
 project(KokkosTutorialTools)
-include(../common.cmake)
+
+# Add a custom module path for find_package
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/../../.cmake)
```

```suggestion
set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/../.cmake)
```

Otherwise it points to a directory outside of the repository.